sway-lib-core/src/ops.sw

```diff
@@ -659,7 +656,7 @@ fn get_word_from_b256(val: b256, offset: u64) -> u64 {
 }
 
 /// Build a single b256 value from 4 64 bit words.
-fn compose(word_1: u64, word_2: u64, word_3: u64, word_4: u64) -> b256 {
+pub fn compose(word_1: u64, word_2: u64, word_3: u64, word_4: u64) -> b256 {
```

Comment:
We're not really pressed for time for this feature, since it can be worked around with hash onion. I would prefer _not_ adding things that aren't ops for primitive types to `core::ops`, especially not at the public interface.

Reply:
fair enough. I linked an issue in the description which will clean this up.

Reply:
This is now being addressed via the addition of a new `core-utils` lib which can be used by both `core` and `std`, while keeping `compose` and `decompose` out of the public API of `std`.